content/docs/v2.3/benchmarks/etcd-2-1-0-alpha-benchmarks.md

```diff
@@ -53,4 +53,4 @@ Bootstrap another machine and use the [boom HTTP benchmark tool][boom] to send r
 | 256               | 256               | all servers        | 3061      | 119.3 |
 
 [boom]: https://github.com/rakyll/boom
-[hack-benchmark]: ../../../hack/benchmark/
+[hack-benchmark]: https://github.com/etcd-io/etcd/tree/master/hack/benchmark
```

Comment:
The URL should be https://github.com/etcd-io/etcd/tree/v2.3.8/hack/benchmark, i.e., replace `master` with `v2.3.8` (here and elsewhere under `/docs/v2.3`).

Reply:
This will go part way into addressing #147.

Reply:
As this is such a big PR, I think I'd like to address this feedback in https://github.com/etcd-io/website/issues/147